source/codegen/proto_helpers.py

```diff
@@ -42,6 +42,8 @@ def get_grpc_type_from_ivi(type, is_array, driver_name_pascal):
     type = 'uint32'
   if 'ViUInt64' in type:
     type = 'uint64'
+  if 'ViUInt8' in type:
```

Comment:
I think we also need support for ViUInt8 array.

Reply:
If it is array, repeated keyword will get added at line 54 below.

Reply:
@doshirohan , ViUInt8 is 8 bit. But for normal ViUInt8, we are using 'uint32' of gRPC. That is okay for just ViUInt8. But when it comes to array, we cannot use repeated uint32 because if you remember arrays are handled as pointers and pointer arithmetic is purely based on number of bits. So array is going to fail with repeated uint32. So we need to use bytes for arrays similar to what is done for[ ViInt8 datatype above. ](https://github.com/ni/grpc-device/blob/main/source/codegen/proto_helpers.py#L29)

Reply:
Makes sense. @ni-pkhare try out things we talked about in Teams and make necessary changes to handle arrays. It would probably be better to go with `bytes` to keep it consistent with `ViInt8` above, but I am still curious to see outcome of trying out with `repeated` and checking if your theory about `uint32` having variable-length encoding still holds true.

Reply:
I have used bytes for ViUInt8 array. I will try with repeated uint32 also just for checking if it also works.